### DISCUSSION TITLE
Translation for "Create option"

### DISCUSSION THREAD

**Discussion by valpuia** [filamentphp/filament#6549] [Help] [answered]
[https://filamentphp.com/docs/2.x/forms/fields#creating-new-records](https://filamentphp.com/docs/2.x/forms/fields#creating-new-records) provides very easy and good form, but one thing I notice is when we hover the `+` icon, it display "Create option" title but I cannot find the way to change this text.. Is there any translation file for this one, because we're handling multiple languages and client asked to change the text to others language also when `hover`

Thanks in advance

**Comment by danharrin** [comment]
Yeah it should be a key in our lang files, you can PR your translation in if you want

**Reply by valpuia** [reply]
Hi @danharrin thanks for the offer, but I am just a junior developers.. all I can do is asking `how to use this` whether that one are already working.. :smile: 

I'll wait for the changes in upcoming version.. thanks

**Reply by danharrin** [reply]
I'm not going to translate it into your language for you. I appreciate you are a junior, but Filament already has a translation system that allows you to change the text, and you will never learn how to use it if you don't try. Ask one of your colleagues about how Laravel translations work.

**Reply by valpuia** [reply]
Hi, thanks for the response, maybe you misunderstood my comment because of my bad english..

What I am trying to said is "I cannot find the `key=>value` for "Create option" in you languages files.

Anyway I'll try to check it out and and If i can change this i'll create a PR

again, thanks for your efforts and helps..

**Reply by danharrin** [reply, marked as answer]
`->createOptionAction(fn ($action) => $action->label('Your label here'))`
